### DISCUSSION TITLE
Update Node.js version on a repository hosted on GitHub and deprecated commands

### DISCUSSION THREAD

**Discussion by D3vil0p3r** [community/community#35814] [Actions] [answered]
Hello,
I have a repository on GitHub that I use as remote repository for Arch Linux custom packages: https://github.com/Athena-OS/athena-repository

The repository is pretty simple and when I push some changes to it, I receive the following warning message:
```
Node.js 12 actions are deprecated. For more information see: https://github.blog/changelog/2022-09-22-github-actions-all-actions-will-begin-running-on-node16-instead-of-node12/. Please update the following actions to use Node.js 16: actions/checkout, actions/checkout
```
Furthermore, I get also other messages related to deprecated commands as:
```
The `save-state` command is deprecated and will be disabled soon. Please upgrade to using Environment Files. For more information see: https://github.blog/changelog/2022-10-11-github-actions-deprecating-save-state-and-set-output-commands/

The `set-output` command is deprecated and will be disabled soon. Please upgrade to using Environment Files. For more information see: https://github.blog/changelog/2022-10-11-github-actions-deprecating-save-state-and-set-output-commands/
```
and despite I followed the provided links, I don't know how to update them because I know that when I run `git push` to my remote repository, an action is run, but I'm a newbie on it and I don't know how I can update these commands and Node.js.

Could you please help me?

Thank you

**Comment by airtower-luna** [comment, marked as answer]
It seems like you don't have any custom workflow there, and just use GitHub's default Pages deployment. In that case the easiest solution is to wait for GitHub to fix that (maybe send a bug report), unless you want to [set up a custom deployment workflow](https://docs.github.com/en/pages/getting-started-with-github-pages/configuring-a-publishing-source-for-your-github-pages-site#publishing-with-a-custom-github-actions-workflow).

**Reply by D3vil0p3r** [reply]
> It seems like you don't have any custom workflow there, and just use GitHub's default Pages deployment. In that case the easiest solution is to wait for GitHub to fix that (maybe send a bug report), unless you want to [set up a custom deployment workflow](https://docs.github.com/en/pages/getting-started-with-github-pages/configuring-a-publishing-source-for-your-github-pages-site#publishing-with-a-custom-github-actions-workflow).

Thank you for the answer. For sending a bug report, should I use GitHub Community website or is there another channel?

**Reply by airtower-luna** [reply]
There's a specific option in the support contact form: https://support.github.com/contact/bug-report
